src/language/class-modifiers.md

```diff
@@ -194,15 +205,16 @@ class MockVehicle          // ERROR: Cannot be implemented
 
 ## `sealed`
 
-Use the `sealed` modifier to create a known, enumerable set of subtypes, so that
-a switch over those types can be statically ensured to be [_exhaustive_][exhaustive].
+To create a known, enumerable set of subtypes, so that a switch over those types
+can be statically ensured to be [_exhaustive_][exhaustive], use the `sealed` modifier.
```

Comment:
**issue:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).
```suggestion
To create a known, enumerable set of subtypes, use the `sealed` modifier.
This ensures [_exhaustive_][exhaustive] coverage of all `cases` included in a `switch` over those types.
```